low-level/actuators/impl/generic/realstor_encl.py

```diff
@@ -593,22 +629,124 @@ def _get_system_info(self):
     def _get_encl_response(self, uri, request_type):
         """ query enclosure and return json data"""
         url = self.rssencl.build_url(uri)
+        response = None

```

Comment:
This needs to be
response = {}

or @ line 703 it may not work as expected

Reply:
Done